SendEmailAdvanced/SendEmailWithManualPollingUsingOperationId/Program.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Azure;
+using Azure.Communication.Email;
+using System;
+using System.Threading.Tasks;
+
+namespace SendEmailWithManualPollingUsingOperationId
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            // This code demonstrates how to send email using Azure Communication Services.
+            var connectionString = "<ACS_CONNECTION_STRING>";
+            var emailClient = new EmailClient(connectionString);
+
+            var sender = "<SENDER_EMAIL>";
+            var recipient = "<RECIPIENT_EMAIL>";
+            var subject = "Send email with manual status polling using operationID";
+
+            var emailContent = new EmailContent(subject)
+            {
+                PlainText = "This is plain text mail send test body \n Best Wishes!!",
+                Html = "<html><body><h1>Quick send email test</h1><br/><h4>Communication email as a service mail send app working properly</h4><p>Happy Learning!!</p></body></html>"
+            };
+
+            var emailMessage = new EmailMessage(sender, recipient, emailContent);
+
+            var emailSendOperation = await emailClient.SendAsync(
+                wait: WaitUntil.Started,
+                message: emailMessage);
+
+            /// Get the OperationId so that it can be used for rehydrating an EmailSendOperation object
+            /// and use that object to poll for the status of the email send operation.
+            var operationId = emailSendOperation.Id;
+            Console.WriteLine($"Email operation id = {operationId}");
+
+            /// Do a bunch of other things here...
+
+            /// Poll for the status of the email send operation using the previous operationId
+            await PollForEmailSendOperationStatus(emailClient, operationId);
+        }
+
```

Comment:
It might be helpful to write a small blurb about what "rehydration" actually is in this LRO context, and why it is necessary. This might just be my lack of technical vocabulary, but if I was just a consumer of this SDK, the "what" and "why" might not be readily clear from the terminology alone. Suggest to add this comment above the method:

```
/// Rehydration refers to the process of creating a new EmailSendOperation object using the operation ID from a previous EmailSendOperation. 
/// This is necessary to continue monitoring the status of the email manually, even after the initial request has been made.
```

Reply:
great suggestion. I'll add that.